src/components/Register/index.jsx

```diff
@@ -0,0 +1,252 @@
+import { Alert, Button, Checkbox, Input } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import { useTranslation } from "react-i18next";
+
+const Register = (props) => {
+  const [t] = useTranslation();
+  const maxLenghtPassword = 8;
+  const [registerInformation, setRegisterInformation] = useState({
+    fullName: "",
+    age: "",
+    email: "",
+    password: "",
+    agree: false,
+  });
+  const [errors, setErrors] = useState({});
+  const [termsOpen, setTermsOpen] = useState(false);
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, registerInformation);
+    newValues[key] = value;
+    setRegisterInformation(newValues);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let { fullName, email, age, password, agree } = registerInformation;
+    let errors = {};
+    if (fullName.trim() === "") {
+      errors["fullName"] = t("register.fullName");
```

Comment:
`setErrors` should be used

Reply:
same should be applied to other cases

Reply:
I see that you use `setErrors `below but you shouldn't manipulate state without setter function here